_includes/urls

```diff
@@ -0,0 +1,5 @@
+{%- if page.version == "master" -%}
+https://projectcalico.org/builds/calicoctl
+{%- else -%}
```

Comment:
would be cool if we could throw a runtime error here if an unexpected value is passed in for component. Maybe put a final `else` that tries to access a negative index of an array or something with a comment? Idk just an idea.